projects/kit/components/rating/rating.template.html

```diff
@@ -0,0 +1,35 @@
+<label
+    #parent
+    class="t-rating"
+    [attr.aria-label]="value + ' Stars'"
+    [class.t-rating--readonly]="readOnly"
+    [class.t-rating--disabled]="disabled"
+>
+    <input
+        type="range"
+        step="1"
+        class="t-rating__input"
+        [max]="max"
+        [ngModel]="value"
+        [attr.tabindex]="readOnly || disabled ? -1 : 0"
```

Comment:
Can we use directive [tuiFocusable](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/cdk/directives/focusable/focusable.directive.ts) here?